sign-up/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import SignUp from "./SignUp";
+import Container from '@mui/material/Container';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
```

Comment:
I don't think we require all the imported components. Can you please check which ones are required and remove the ones that are not required?